requirements.txt

```diff
@@ -0,0 +1,19 @@
+aniso8601==9.0.1 ; python_version >= "3.9" and python_version < "4.0"
+attrs==23.2.0 ; python_version >= "3.9" and python_version < "4.0"
+blinker==1.8.2 ; python_version >= "3.9" and python_version < "4.0"
+click==8.1.7 ; python_version >= "3.9" and python_version < "4.0"
+colorama==0.4.6 ; python_version >= "3.9" and python_version < "4.0" and platform_system == "Windows"
```

Comment:
(On the other hand, I'm assuming this just means we don't need colorama if we're not Windows, so that seems reasonable and potentially important!)

Reply:
Yeah, I understand colorama to have windows-specific improvements that aren't necessary on other platforms :D